compat/src/suspense.js

```diff
@@ -15,20 +15,21 @@ export function catchRender(error, component) {
 	return false;
 }
 
-function removeDom(vnode) {
-	let tmp = vnode._component;
-	if (tmp != null) {
-		if (tmp = tmp._prevVNode) removeDom(tmp);
-	}
-	else if ((tmp = vnode._children) && vnode._lastDomChild) {
-		for (let i = 0; i < tmp.length; i++) {
-			if (tmp[i]) removeDom(tmp[i]);
+function removeDom(children) {
+	// TODO: We need to recurse down the tree cuz _dom pointers aren't bubbled up
+	// when a component suspends. See comment about "early exit" in diff/index.js
+	// for a possible resolution
+	for (let i = 0; i < children.length; i++) {
```

Comment:
@sventschui Can you take a look particularly at these suspense changes, and double check that they makes sense to you?

Reply:
Looks good 👍